rails/bad-paths-can-trigger-invalid_cross_origin_request.md

```diff
@@ -0,0 +1,38 @@
+# Bad JS paths can trigger `InvalidCrossOriginRequest`
+
+Rails v4.1.6
+
+In a controller using [`protect_from_forgery`](http://guides.rubyonrails.org/security.html#csrf-countermeasures),
+moving (but not otherwise renaming) a JS file referenced by that
+controller's view(s) without updating the view(s) can trigger an
```

Comment:
Could we simplify this paragraph by just saying this error can be triggered by including an incorrect script path?